website/description_issues.qmd

```diff
@@ -40,3 +43,38 @@ Document all non-obvious acronyms in the [cran-comments.md](https://usethis.r-li
 ### Details
 
 Most acronyms that are not widely known should be explained. For example, you don’t have to explain *OLS*, *SEO*, or *DNA* but explanations should be provided for: *MEFM* or *OCCDS*. If you are unsure of the acronyms common knowledge, please document all acronyms in the package.
+
+
+---
+
+
+# LICENSE files
+
+## Problem
+
+A file is referenced in the DESCRIPTION field for License when it's not required or the full license has been copied and submitted in the LICENSE file.
```

Comment:
I am not a native English speaker, as many R users, and this sentence is a bit hard to understand to me. 

```suggestion
Some licenses do not need to be added verbatim, instead they are referenced in the DESCRIPTION file and the CRAN template is added to the package. You can find the template at the [CRAN website](https://cran.r-project.org/web/licenses/).
```

Perhaps also explain that if for other purposes they want to add the full text they can add it to the .Rbuildignore file so that when it is build and sent to CRAN the full license text won't be included.